com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDDynamicResolutionPlatformCapabilities.cs

```diff
@@ -4,29 +4,19 @@ namespace UnityEngine.Rendering.HighDefinition
 {
     public static class HDDynamicResolutionPlatformCapabilities
     {
-        public enum Flag
-        {
-            DLSSDetected
-        }
+        /// <summary>
+        /// True if the render pipeline detected DLSS capable platform. False otherwise.
+        /// </summary>
+        public static bool DLSSDetected { get { return m_DLSSDetected; } }
 
-        public static bool GetFlag(Flag flag)
-        {
-            return (s_FeatureFlags & (1 << (int)flag)) != 0;
-        }
+        /// <summary>
+        /// User only variable, to control wether DLSS is on or off programmatically / from user code..
+        /// </summary>
+        public static bool DLSSEnabled { set { m_DLSSEnabled = value; } get { return m_DLSSEnabled && m_DLSSDetected; } }
```

Comment:
I wonder if we want to split the set and the get here, DLSSEnabled set to true can retrieve immediately false if dlss is not detected, splitting them would give us the option to give explicit naming

Reply:
This has been fixed